code/src/components/StartPage.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react'
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux'
+import Lottie from 'lottie-react';
+import { game, getGameStarted } from '../reducers/game'
+import { StyledButton } from './GlobalStyles';
+import animationData from '../lotties/rabbit.json';
+
+const StartWrapper = styled.div`
+display: flex;
+flex-direction: column;
+justify-content: space-evenly;
+height: 100vh;
+align-items: center;
+background: rgba(213, 146, 131, 0.59);
+border-radius: 16px;
+box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+backdrop-filter: blur(3.9px);
+-webkit-backdrop-filter: blur(3.9px);
+border: 1px solid rgba(213, 146, 131, 0.3);
```

Comment:
When looking at inspect, it only looks like the yellow border is showing up. Did you mean to have two borders?